include/printk.h

```diff
@@ -0,0 +1,22 @@
+#ifndef __PRINTK_H__
+#define __PRINTK_H__
+
+#include <types.h>
+
+#ifdef DEBUG
+
+void print(const char * unused);
+void print_p(const void * unused);
```

Comment:
What style are you going for?  Elsewhere, there is space after `*`

Reply:
I guess that would be, "I missed that sh^Htuff" style